"""Plugin-related exceptions."""

from __future__ import annotations


class PluginError(Exception):
    """Raised when plugin operations fail."""

    pass
